### PR TITLE
Modify key wording

### LINKED ISSUE #1464: Keys tab under Manage Profile should display information about a user's key and origin

### PR DESCRIPTION
**Pull request by nellshamrell**
Fixes #1464

### CODE REVIEW

File: src/supermarket/spec/models/user_spec.rb [hunk 1]
```diff
@@ -551,4 +551,47 @@
       expect(user.reload.install_preference).to eql('knife')
     end
   end
+
+  describe '#public_key_signature' do
```
Comment (robbkidd) [inline src/supermarket/spec/models/user_spec.rb]: OOOoookay. I've got a more reliable way to check all this.

```ruby
def public_key_signature
  # Inspired by https://stelfox.net/blog/2014/04/calculating-rsa-key-fingerprints-in-ruby/
  # Verifiable by an end-user either:
  #   with private key: openssl rsa -in private_key.pem -pubout -outform DER | openssl md5 -c
  #   with public key: openssl rsa -in public_key.pub -pubin -outform DER | openssl md5 -c
  key_in_der_format = OpenSSL::PKey::RSA.new(public_key).to_der
  OpenSSL::Digest::MD5.hexdigest(key_in_der_format).scan(/../).join(':')
end
```

Showing my work:

```shell
# using openssl in the shell
src/supermarket/spec/support/key_fixtures > openssl rsa -in valid_private_key.pem -pubout -outform DER | openssl md5 -c
writing RSA key
0f:d2:d4:d9:76:14:ab:8e:bd:67:87:d5:d6:a7:24:29

src/supermarket/spec/support/key_fixtures > openssl rsa -in valid_public_key.pub -pubin -outform DER | openssl md5 -c
writing RSA key
0f:d2:d4:d9:76:14:ab:8e:bd:67:87:d5:d6:a7:24:29
```

```ruby
# using OpenSSL gem in pry
[3] pry(main)> def public_key_signature(public_key)
[3] pry(main)*   key_in_der_format = OpenSSL::PKey::RSA.new(public_key).to_der
[3] pry(main)*   OpenSSL::Digest::MD5.hexdigest(key_in_der_format).scan(/../).join(':')
[3] pry(main)* end
=> :public_key_signature

[4] pry(main)> valid_public_key = File.read('spec/support/key_fixtures/valid_public_key.pub')
=> "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0ueqo76MXuP6XqZBILFz\niH/9AI7C6PaN5W0dSvkr9yInyGHSz/IR1+4tqvP2qlfKVKI4CP6BFH251Ft9qMUB\nuAsnlAVQ1z0exDtIFFOyQCdR7iXmjBIWMSS4buBwRQXwDK7id1OxtU23qVJv+xwE\nV0IzaaSJmaGLIbvRBD+qatfUuQJBMU/04DdJIwvLtZBYdC2219m5dUBQaa4bimL+\nYN9EcsDzD9h9UxQo5ReK7b3cNMzJBKJWLzFBcJuePMzAnLFktr/RufX4wpXe6XJx\noVPaHo72GorLkwnQ0HYMTY8rehT4mDi1FI969LHCFFaFHSAaRnwdXaQkJmSfcxzC\nYQIDAQAB\n-----END PUBLIC KEY-----\n"

[5] pry(main)> public_key_signature valid_public_key
=> "0f:d2:d4:d9:76:14:ab:8e:bd:67:87:d5:d6:a7:24:29"
```

Working on an update to this PR with all that.

**Comment by robbkidd** [comment]
Also worked over the Keys tab content: 

screen shot 2016-12-08 at 3 34 41 pm

**Comment by robbkidd** [comment]
Pending re-review, I think we can squoosh and send to the pipeline.

**Comment by nellshamrell** [comment]
👍

**Pull request merged**